src/main/scala/firrtl/Emitter.scala

```diff
@@ -469,31 +514,39 @@ class VerilogEmitter extends SeqTransform with Emitter {
     * @param writer       where rendering will be placed
     * @return             the render reference
     */
-  def getRenderer(descriptions: Seq[DescriptionAnnotation],
-                  m: Module,
-                  moduleMap: Map[String, DefModule])(implicit writer: Writer): VerilogRender = {
+  def getRenderer(descriptions: Seq[DescriptionAnnotation], m: Module, moduleMap: Map[String, DefModule])(implicit
+    writer:                     Writer
+  ): VerilogRender = {
```

Comment:
There are definitely some things we should improve lol. Wow.

Reply:
In fact this is actively worse, interesting that sometimes on defs it changes to 1 line per argument (like `addFormalStatement` below) which is definitely better and in this case it does whatever this is.

Reply:
Note that this option improves things:
```
newlines.implicitParamListModifierPrefer = before
```
I have further improvements that will be PR-ed shortly (and rebased on master)